### ISSUE TITLE
[Bug] error: git cmd when following docs

### ISSUE THREAD

**Issue by tekumara** [ray-project/kuberay#288]
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/kuberay/issues) and found no similar issues.

### KubeRay Component

ray-operator

### What happened + What you expected to happen

Following the [docs here](https://ray-project.github.io/kuberay/deploy/helm/#installing-the-chart):
```
$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0" 

error: git cmd = '/usr/local/bin/git fetch --depth=1 origin v0.3.0': exit status 128
```

### Reproduction script

See above

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by Jeffwan** [comment]
Thanks for reporting the issue and I will fix the doc. Please use latest or v0.2.0.

**Comment by tekumara** [comment]
This still happens when [following the docs](https://ray-project.github.io/kuberay/deploy/installation/#nightly-version) for the nightly version:

```
export KUBERAY_VERSION=master
kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=${KUBERAY_VERSION}&timeout=90s"
```

I think the problem is the `timeout=90s` param is not supported.

**Comment by kevin85421** [comment]
* I can reproduce this issue with `kubectl v1.21.0` 
```console
$ kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=${KUBERAY_VERSION}&timeout=90s"
error: git cmd = '/usr/local/bin/git fetch --depth=1 origin v0.3.0&timeout=90s': exit status 128

$ kubectl version --client=true --short=true
Client Version: v1.21.0
```

* Fix this issue by updating `kubectl` to v1.22.0+
```console
$ kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources?ref=${KUBERAY_VERSION}&timeout=90s"
namespace/ray-system created
customresourcedefinition.apiextensions.k8s.io/rayclusters.ray.io created
customresourcedefinition.apiextensions.k8s.io/rayjobs.ray.io created
customresourcedefinition.apiextensions.k8s.io/rayservices.ray.io created

$ kubectl version --client=true --short=true
Client Version: v1.22.0
```

* You can upgrade your `kubectl` by the following commands
```bash
curl -LO "https://dl.k8s.io/release/v1.22.0/bin/darwin/amd64/kubectl"
chmod +x kubectl
sudo cp kubectl $YOUR_PATH
```

I will update the document. Thank you for reporting this issue!
